sheets/xinput

```diff
@@ -25,7 +25,7 @@ xinput --set-prop 8 157 1.000000 0.000000 0.000000 0.000000 1.000000 0.000000 0.
 xinput disable [touchscreen XID]
 
 # Disable middle mouse button click
-# Numbers are what you do with the first, second, and third (ie, left, middle, right) mouse buttons.
+# Numbers are what you do with the first, second, and third (i.e., left, middle, right) mouse buttons.
```

Comment:
I'd say both are accepted forms, here. You'll occasionally see either one used. "I.E" means "id est" (apparently meaning: "that is"), which you could argue, being two separate words (I have no knowledge of that language) warrants a capitalized abbreviation.